.github/workflows/release-please.yml

```
@@ -74,7 +74,7 @@ jobs:
         run: |
           COMMIT_TAG=${{ steps.version.outputs.semver }}
           # need to specify at least one version in versions.json for it to build
-          echo "[\n  \"$(./scripts/get_current_version.sh)\"\n]" > versions.json
+          echo "[ \"$(./scripts/get_current_version.sh)\" ]" > versions.json
```

Use `printf` to correctly format:
```suggestion
          printf "[\n  \"$(./scripts/get_current_version.sh)\"\n]" > versions.json
```
(Previously tested on zsh locally which is why it worked)